src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -1163,6 +1162,14 @@ private void ExecuteAndVerify(SqlCommand cmd, StePermutation tvpPerm, object[][]
             {
                 conn.Open();
                 cmd.Connection = conn;
+                if (DataTestUtility.IsNotAzureServer())
+                {
+                    // Chose the 2628 error message instead of 8152 in SQL Server 2016 & 2017
+                    using (SqlCommand cmdFix = new SqlCommand("DBCC TRACEON(460)", conn))
+                    {
+                        cmdFix.ExecuteNonQuery();
+                    }
+                }
                 //                cmd.Transaction = conn.BeginTransaction();
```

Comment:
IF this not needed, please remove the commented out line.

Reply:
Thank you for point to this.
It was here from the first. I intentionally didn't remove it to get its permission here.

Reply:
@JRahnama is one of the MS team on the library so if he says it can be removed it should be fine to do so. It's the pesky contributors like me you need to be careful of 😁

Reply:
@Wraith2 (Davoud is the newest member of our team. 😄)